rest-api/test/unit_test/api_test/patient_status_api_test.py

```diff
@@ -63,16 +63,22 @@ def test_patient_status_udpated(self):
     # insert patient status
     url = os.path.join('PatientStatus', 'P{0}'.format(self.participant.participantId), 'Organization',
                        'PITT_BANNER_HEALTH')
-    self.send_post(url, data, expected_status=httplib.CREATED)
+    resp = self.send_post(url, data, expected_status=httplib.CREATED)
 
     data['authored'] = '2019-04-27T16:32:01'
     data['comment'] = 'saw patient at new site'
     data['site'] = 'hpo-site-bannerphoenix'
 
-    self.send_put(url, data, expected_status=httplib.OK)
+    resp = self.send_put(url, data, expected_status=httplib.OK)
+    self.assertDictContainsSubset(data, resp)
 
     # Get record and test that our test_data dict is in the resp.response dict.
     resp = self.send_get(url)
     self.assertDictContainsSubset(data, resp)
 
-  # TODO: When new style history tables and triggers have been added to unit tests, test history URL.
+    # TODO: When new style history tables and triggers have been added to unit tests, test history URL.
```

Comment:
Its still not possible to test against the new style history tables. I'm creating a ticket for that.